hs_file_types/views.py

```diff
@@ -538,9 +546,16 @@ def add_metadata_element(request, hs_file_type, file_type_id, element_name, **kw
                               'element_id': element.id,
                               'metadata_status': metadata_status}
 
+        if logical_file.type_name() == "NetCDFLogicalFile":
+            logical_file.metadata.is_update_file = True
+            logical_file.metadata.save()
+
         if logical_file.type_name() == "TimeSeriesLogicalFile":
-            ajax_response_data['is_dirty'] = logical_file.metadata.is_dirty
+            logical_file.metadata.is_update_file = True
+            logical_file.metadata.save()
+            ajax_response_data['is_update_file'] = logical_file.metadata.is_update_file
             ajax_response_data['can_update_sqlite'] = logical_file.can_update_sqlite_file
```

Comment:
Same comment as above to combine two conditional checks into one.

Reply:
Fixed